WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -0,0 +1,79 @@
+package org.wordpress.android.datasets;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import org.wordpress.android.WordPress;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.models.Role;
+import org.wordpress.android.util.SqlUtils;
+
+public class PeopleTable {
+    public static final String PEOPLE_TABLE = "people";
+
+    private static SQLiteDatabase getReadableDb() {
+        return WordPress.wpDB.getDatabase();
+    }
+    private static SQLiteDatabase getWritableDb() {
+        return WordPress.wpDB.getDatabase();
+    }
+
+    public static void createTables(SQLiteDatabase db) {
+        db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
+                + "person_id               INTEGER PRIMARY KEY DEFAULT 0,"
+                + "user_name               TEXT,"
+                + "first_name              TEXT,"
+                + "last_name               TEXT,"
+                + "display_name            TEXT,"
+                + "avatar_url              TEXT,"
+                + "role                    TEXT)");
+    }
+
+    private static void dropTables(SQLiteDatabase db) {
```

Comment:
Hey @oguzkocer , should I assume that this method will be used in some subsequent commits/PR? If yes, just ignore this comment. If not, can this be an indication of a `reset()` method missing maybe?